openapi/events-api.yaml

```diff
@@ -2114,6 +2116,8 @@ components:
 
             PHQ Labels leverage AI and classifier models. Currently available for conferences, expos, festivals, community and performing-arts categories. Legacy labels are available in a separate field to preserve backwards compatibility.
 
+            This field will only be present if you have access to it in your subscription.
```

Comment:
PHQ Labels although technically can be turned off, it's not something we plan on turning off as standard practice so this line can be removed.